specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13632,44 +13632,24 @@
     "MSDeploy": {
       "description": "MSDeploy ARM PUT information",
       "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
       "properties": {
-        "addOnPackages": {
-          "description": "List of Add-On packages. Add-On packages implicitly enable the Do Not Delete MSDeploy rule.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/MSDeployCore"
-          }
-        },
-        "packageUri": {
-          "description": "Package URI",
-          "type": "string"
-        },
-        "connectionString": {
-          "description": "SQL Connection String",
-          "type": "string"
-        },
-        "dbType": {
-          "description": "Database Type",
-          "type": "string"
-        },
-        "setParametersXmlFileUri": {
-          "description": "URI of MSDeploy Parameters file. Must not be set if SetParameters is used.",
-          "type": "string"
-        },
-        "setParameters": {
```

Comment:
@EricSten-MSFT Can you let us know why you chose array over dictionary? Since this is critical to the fluent SDK team and the ARM deploy above says it should be a dictionary, I will change it right now. @EricSten-MSFT Let me know if there is any reason why we should not.